container/package.json

```diff
@@ -62,6 +62,9 @@
     "cli-color": "^2.0.4",
     "github-api": "^3.4.0"
   },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-x64-gnu": "*"
```

Comment:
what is optionalDep used for 🤔  which other one needs it

Reply:
Without it bundle job for container was failing:
https://github.com/SAP/luigi/actions/runs/11071425700/job/30763262753